### ISSUE TITLE
when to stop training deeplab

### ISSUE THREAD

**Issue by essalahsouad** [tensorflow/models#6924]
**### System information**
**What is the top-level directory of the model you are using:**
https://github.com/tensorflow/models/tree/master/research/deeplab
 ** Have I written custom code (as opposed to using a stock example script provided in TensorFlow):**
No
**OS Platform and Distribution (e.g., Linux Ubuntu 16.04):**ubuntu 18.04
**TensorFlow installed from (source or binary):**binary
**TensorFlow version (use command below):**1.13.1 CPU
**Bazel version (if compiling from source):**N/A
**CUDA/cuDNN version:**9.2/7.1
**GPU model and memory:**GeForce GTX 970M 

### Exact command to reproduce:
`python train.py \ --logtostderr \ --train_split="train" \ --model_variant="xception_65" \ --atrous_rates=6 \ --atrous_rates=12 \ --atrous_rates=18 \  --output_stride=16 \ --decoder_output_stride=4 \ --train_crop_size=609 \ --train_crop_size=609 \ --train_batch_size=1 \  --training_number_of_steps=7000 \ --fine_tune_batch_norm=False \ --`
data=3400 images (604*604+400*300)
loss=0.2
Knowing that the loss slowly changing during the training it start with 4 and on step 700 is 0.2
and predicted mask is not good  enough
